website/docs/r/firewall.html.markdown

```diff
@@ -106,6 +116,18 @@ A `management_ip_configuration` block supports the following:
 
 -> **NOTE** The Public IP must have a `Static` allocation and `Standard` sku.
 
+---
+
+A `virtual_hub_setting` block supports the following:
+
+* `virtual_hub_id` - (Required) Specifies the ID of the Virtual Hub where the Firewall resides in.
+
+* `public_ip_count` - (Optional) Specifies the amount of public IP to assign for the Firewall. Defaults to `1`.
+
+* `public_ip_addresses` - (Optional) A list of public IP addresses associated with the Firewall.
+
+-> **NOTE** In most cases, users do not need to specify this argument, except when scaling down the public IPs (by changing the `public_ip_count`), this argument allows users to specify the existing IPs to be retained. Otherwise, the first existing `public_ip_count` public ip addresses will be retained.
```

Comment:
The last change here will change the meaning a bit, it will affect the first existing N public ip addresses actually, where N equals to `public_ip_count`.

Reply:
I'm not sure how better to word it as i still don't fully understand what it is doing :/

Reply:
This means when you scale down the public IPs of the firewall, say from (pip1, pip2, pip3) to only one pip. If you simply change the `public_ip_count` from 3 to 1. Then it ends up with the pip1. However, if you intend to keep the pip3, then you can specify it using this property, say `public_ip_addresses = [pip3]`.